src/ngx_http_lua_headers_out.c

```diff
@@ -499,47 +501,26 @@ ngx_http_lua_set_output_header(ngx_http_request_t *r, ngx_http_lua_ctx_t *ctx,
         return NGX_ERROR;
     }
 
+    ngx_strlow(key.data, key.data, key.len);
```

Comment:
so, are we always setting the lowercase keys now?

if yes, I think don't think it's a good idea.

Reply:
I think we can add a hash_find_lc function,In some cases, it can be avoided alloc memory to convert lower case

Reply:
I think we can add a hash_find_lc function,In some cases, it can be avoided alloc memory to convert lower case